src/Microsoft.VisualStudio.SlnGen.UnitTests/TempDirectory.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation.
+//
+// Licensed under the MIT license.
+
+using System;
+using System.IO;
+
+namespace Microsoft.VisualStudio.SlnGen.UnitTests
+{
+    internal class TempDirectory : IDisposable
```

Comment:
The test base class `TestBase` already has an implementation to create a temp folder for each test.  You can access the `TestRootPath` folder to use it.

https://github.com/microsoft/slngen/blob/main/src/Microsoft.VisualStudio.SlnGen.UnitTests/TestBase.cs#L32

Reply:
Thank you. I've pushed another commit to resolve this.